generate-wrapper.py

```diff
@@ -59,43 +101,60 @@ def visit_FuncDecl(self, node):
 
        if node.args:
            for p in node.args.params:
-               args.append(typename(p.type))
+               args.append((p.name, typename(p.type)))
```

Comment:
I added the name of each arg to the `args` array, so I can generate the TypeScript function signatures with the correct argument names.